openedx/core/djangoapps/course_groups/cohorts.py

```diff
@@ -369,12 +358,11 @@ def add_user_to_cohort(cohort, username_or_email):
             "user_id": user.id,
             "cohort_id": cohort.id,
             "cohort_name": cohort.name,
-            "previous_cohort_id": previous_cohort_id,
-            "previous_cohort_name": previous_cohort_name,
+            "previous_cohort_id": membership.previous_cohort_id,
+            "previous_cohort_name": membership.previous_cohort_name,
         }
     )
```

Comment:
Spoke offline with @explorerleslie, we'll be emitting the same event, with previous cohort information, after the `save()` call returns.